policyengine_us/parameters/gov/irs/deductions/itemized/reduction/rate/base.yaml

```diff
@@ -2,8 +2,10 @@ description: IRS multiplies the excess of the total and reduced itemized deducti
 values:
   2017-01-01: 0.8
   2018-01-01: 0
```

Comment:
instead of this could we nullify the limitation during tcja? seems more aligned to the irc